easybuild/easyconfigs/a/awscli/awscli-2.1.37-GCCcore-10.2.0.eb

```diff
@@ -0,0 +1,70 @@
+easyblock = 'PythonBundle'
+
+name = 'awscli'
+version = '2.1.37'
+
+homepage = 'https://pypi.python.org/pypi/awscli'
+description = 'Universal Command Line Environment for AWS'
+
+toolchain = {'name': 'GCCcore', 'version': '10.2.0'}
+
+builddependencies = [
+    ('binutils', '2.35'),
+]
+
+dependencies = [
+    ('Python', '3.8.6'),
+    ('PyYAML', '5.3.1'),
+]
+
+use_pip = True
+
+exts_default_options = {'source_urls': [PYPI_SOURCE]}
+
+exts_list = [
+    ('jmespath', '0.10.0', {
+        'checksums': ['b85d0567b8666149a93172712e68920734333c0ce7e89b78b3e987f71e5ed4f9'],
+    }),
+    ('botocore', '1.20.53', {
+        'source_tmpl': '2c5e5782c4.tar.gz',
+        'source_urls': ['https://github.com/boto/botocore/archive/'],
+        'checksums': ['a8d6340ded75ce69bc10db9c5b37d0ef443fbad417a366c6a702b4a3d4594ad4'],
+    }),
+    ('docutils', '0.15.2', {
```

Comment:
again, 0.16 already part of Python 3.8.6?

Reply:
setup.py is very strict